src/Sylius/Bundle/AddressingBundle/Doctrine/ORM/ZoneRepository.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AddressingBundle\Doctrine\ORM;
+
+use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
+use Sylius\Component\Addressing\Repository\ZoneRepositoryInterface;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ZoneRepository extends EntityRepository implements ZoneRepositoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function findOneByName($name)
+    {
+        return $this->createQueryBuilder('o')
+            ->addSelect('members')
```

Comment:
We should use singular form here, because later you reference that as ``members.name`` for example, etc. Unless we want to differentiate between to-one / to-many relations, but that would require changing convention everywhere. I am fine with merging this as is, but let's discuss it tomorrow in the office.